src/content/docs/browser/new-relic-browser/browser-apis/setcustomattribute.mdx

```diff
@@ -32,14 +32,22 @@ Adds a user-defined attribute name and value to subsequent events on the page.
 
 Agent version [nr-593](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes) or higher.
 
+For persist param or `null` value support, agent version 1.230.0 or higher is required.
```

Comment:
@cwli24 is "persist param" the same as the `persist` parameter? If so, could you spell that out: "For `persist` parameter or `null` value support, agent version 1.230.0 or higher is required.